packages/core/src/exchanges/cache.ts

```diff
@@ -59,7 +59,7 @@ export const cacheExchange: Exchange = ({ forward, client }) => {
         const cachedResult = resultCache.get(operation.key);
 
         if (cachedResult) {
-          client.eventTarget.dispatchEvent({
+          client.eventTarget!.dispatchEvent({
```

Comment:
Seems like we should enforce that `eventTarget` is defined and assume that it does, since the babel plugin takes care of creation for us?

Reply:
I think this is fine since this essentially comes down to TS not knowing that this is undefined in prod and that the function can't be called in prod. I could also just strip everything from the class instead if you like?

Reply:
Nah, I think this is a minor detail so either is actually fine on second thought :+1: